### PR TITLE
github/labeler: label manpages changes with related component labels

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -111,6 +122,7 @@ CI:
 
 rbd:
   - doc/rbd/**
+  - doc/man/8/rbd*
```
Comment (dillaman) [inline .github/labeler.yml:125]: Nit: I would just move this up one line so that it remains in lexicographic order.

**Review by dillaman — APPROVED** [review]

**Review by epuertat — APPROVED** [review]
👍🏼

**Comment by epuertat** [comment]
Just checking whether the autolabel removal was fixed...

**Review by jdurgin — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (tchaikov) [reply .github/labeler.yml:125]: updated and repushed.

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**